pkg/manifests/manifests.go

```diff
@@ -372,6 +377,10 @@ func (f *Factory) KubeStateMetricsDeployment() (*appsv1.Deployment, error) {
 		d.Spec.Template.Spec.NodeSelector = f.config.KubeStateMetricsConfig.NodeSelector
 	}
 
+	if len(f.config.KubeStateMetricsConfig.PriorityClassName) > 0 {
```

Comment:
for consistency, could we change all of these string comparisons from `len(x) > 0` to `x != ""`?